src/applications/IsIdentifiableReviewer/MainWindow.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using IsIdentifiableReviewer.Out;
+using Microservices.IsIdentifiable.Rules;
+using MongoDB.Bson;
+using NPOI.OpenXmlFormats.Dml.Diagram;
+using Terminal.Gui;
+using Attribute = Terminal.Gui.Attribute;
+
+namespace IsIdentifiableReviewer
+{
+    class MainWindow : View
+    {
+        private readonly List<Target> _targets;
+
+        public Target CurrentTarget { get; set; }
+        public ReportReader CurrentReport { get; set; }
+
+        public RuleGenerator Generator { get; set; } = new RuleGenerator(new FileInfo("NewRules.yaml"));
+
+        public RowUpdater Updater { get; set; } = new RowUpdater();
+
+        public int DlgWidth = 78;
+        public int DlgHeight = 18;
+        public int DlgBoundary = 2;
+        private ValuePane _valuePane;
+        private Label _info;
+
+        public MainWindow(List<Target> targets)
+        {
+            _targets = targets;
+            X = 0;
+            Y = 1;
+            Width = Dim.Fill();
+            Height = Dim.Fill();
+            
+            var top = Application.Top;
+
+            var menu = new MenuBar (new MenuBarItem [] {
+                new MenuBarItem ("_Game (F9)", new MenuItem [] {
```

Comment:
Game??? Should probably be File or Report, something like that?

Reply:
Mistakes were made! (https://github.com/tznind/Wanderer/blob/6bd59b861024a7b48a31273e18bcf5631a35973c/Game/UI/MainWindow.cs#L52)